crates/ibc/src/core/ics04_channel/packet.rs

```diff
@@ -101,6 +103,18 @@ impl From<Sequence> for u64 {
     }
 }
 
+impl TryFrom<Sequence> for Vec<u8> {
+    type Error = PacketError;
+
+    fn try_from(seq: Sequence) -> Result<Self, Self::Error> {
+        let mut value = Vec::new();
+        u64::from(seq)
+            .encode(&mut value)
+            .expect("buffer size too small");
+        Ok(value)
+    }
+}
+
```

Comment:
I don't think we should have this here; the protobuf way is one way to encode a `u64` to a byte array; this suggests that it is "the right way to do it".

Reply:
we should also return an error instead of the `expect()`

Reply:
fcd8b791d29a82e838e7724c58ae4e804502bc29